auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SaveSmartLock.java

```diff
@@ -142,7 +142,7 @@ public void saveCredentialsOrFinish(FirebaseUser firebaseUser,
         // Build credentials client and kick off the save
         mCredentialsClient = Credentials.getClient(getActivity());
         Credential credential = CredentialsUtil.buildCredential(
-                mEmail, mName, mPassword, mProfilePictureUri, mResponse);
+                mEmail, mPassword, mName, mProfilePictureUri, mResponse);
```

Comment:
This is the scariest thing.  Need to figure out what test should have failed and make it MUCH LOUDER.